src/test/java/com/google/cloud/compute/v1/it/BaseTest.java

```diff
@@ -140,27 +138,4 @@ static void cleanUpNetwork(Network network) throws IOException {
 
     waitForOperation(networkClient.deleteNetwork(network.getSelfLink()));
   }
-
-  static void cleanUpNetworks() throws IOException {
-    NetworkSettings networkSettings =
-        NetworkSettings.newBuilder().setCredentialsProvider(credentialsProvider).build();
-    NetworkClient networkClient = NetworkClient.create(networkSettings);
-
-    // clean up resources older than 1 hour
-    Calendar calendar = Calendar.getInstance();
-    calendar.add(Calendar.HOUR_OF_DAY, -1);
-    Timestamp cutoff = Timestamp.of(calendar.getTime());
```

Comment:
This `cleanUpNetworks` was here to delete old networks that are not cleaned up in previous test runs. We have a hard limit of ~10 networks on the project which cannot be changed.